src/main/scala/elevate/rise/rules/algorithmic.scala

```diff
@@ -81,30 +120,77 @@ object algorithmic {
       case App(Id(), arg) => Success(app(transpose >> transpose, arg) :: e.t)
       case _              => Failure(createTransposePair)
     }
-    override def toString = "createTransposePair"
+    override def toString: String = "createTransposePair"
   }
 
-  def `_-> T >> T`: Strategy[Rise] = idAfter `;` createTransposePair
-
   def removeTransposePair: Strategy[Rise] = `T >> T -> `
   case object `T >> T -> ` extends Strategy[Rise]  {
     def apply(e: Rise): RewriteResult[Rise] = e match {
       case App(Transpose(), App(Transpose(), x)) => Success(x :: e.t)
       case _                                     => Failure(removeTransposePair)
     }
-    override def toString = "createTransposePair"
+    override def toString: String = "createTransposePair"
   }
 
   // slideSeq fusion
   import rise.OpenCL.primitives._
   import rise.OpenCL.TypedDSL._
 
-  def slideSeqFusion: Strategy[Rise] = `slideSeq(f) >> map(g) -> slideSeq(f >> g)`
```

Comment:
why remove the "specification"?

Reply:
because we're not using these functions anywhere, if we want a "specification" we should add them as comments or scala doc but I think having another function as alias is just cluttering up the code